typings/index.d.ts

```diff
@@ -57,61 +71,147 @@ export interface QueryByAPI {
   ) => Array<ReactTestInstance> | [];
 
   // Unsafe aliases
-  UNSAFE_queryByType: <P>(type: React.ComponentType<P>) => ReactTestInstance | null,
-  UNSAFE_queryAllByType: <P>(type: React.ComponentType<P>) => Array<ReactTestInstance> | [],
-  UNSAFE_queryByProps: (props: Record<string, any>) => ReactTestInstance | null,
-  UNSAFE_queryAllByProps: (props: Record<string, any>) => Array<ReactTestInstance> | [],
+  UNSAFE_queryByType: <P>(
+    type: React.ComponentType<P>
+  ) => ReactTestInstance | null;
+  UNSAFE_queryAllByType: <P>(
+    type: React.ComponentType<P>
+  ) => Array<ReactTestInstance> | [];
+  UNSAFE_queryByProps: (props: Record<string, any>) => ReactTestInstance | null;
+  UNSAFE_queryAllByProps: (
+    props: Record<string, any>
+  ) => Array<ReactTestInstance> | [];
 }
 
-type GetReturn = ReactTestInstance;
-type GetAllReturn = Array<ReactTestInstance>;
-type QueryReturn = ReactTestInstance | null;
-type QueryAllReturn = Array<ReactTestInstance> | [];
+export interface WaitForOptions {
+  timeout: number;
+  interval: number;
+}
+
+export interface FindByAPI {
```

Comment:
I don't think we need to export this interface, what for?

Reply:
Same happens for GetByAPI & QueryByAPI interfaces. Do you things it's good idea to remove all of these exports?

Reply:
If we don't use them, then let's remove. If they're useful for somebody, they'll send a PR to export it :)

Reply:
I've removed exports for all three of them